variables.tf

```diff
@@ -1,7 +1,7 @@
-# Example, compulsory input variable
-variable "group" {
+variable "group_email" {
   description = "Group ID"
   type        = string
+  default     = "example-group@google.com"
```

Comment:
I think we shouldn't provide default at all

Reply:
ok